src/pepr/uds-operator-config/templates/clusterconfig.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+apiVersion: uds.dev/v1alpha1
+kind: ClusterConfig
+metadata:
+  name: uds-cluster-config
+  namespace: pepr-system
+  labels:
+    {{- include "uds-operator-config.labels" . | nindent 4 }}
+spec:
+  attributes:
+    clusterName: {{ .Values.cluster.attributes.UDS_CLUSTER_NAME | quote }}
+    tags:
+{{- if not (eq (typeOf .Values.cluster.attributes.UDS_CLUSTER_TAGS) "string") }}
+{{ toYaml .Values.cluster.attributes.UDS_CLUSTER_TAGS | nindent 6 }}
+{{- end }}
+  networking:
+    kubeapiCIDR: {{ .Values.cluster.networking.KUBEAPI_CIDR | default .Values.operator.KUBEAPI_CIDR | quote }}
+    kubenodeCIDRS:
+{{- if .Values.cluster.networking.KUBENODE_CIDRS }}
+{{ toYaml .Values.cluster.networking.KUBENODE_CIDRS | indent 6 }}
+{{- else if .Values.operator.KUBENODE_CIDRS }}
+{{- range $index, $cidr := .Values.operator.KUBENODE_CIDRS }}
+      - {{ $cidr | quote }}
+{{- end }}
```

Comment:
I'll probably have to test this out locally so putting a placeholder comment for now but this templating looks off to me. I'd expect that we need to split on `,` here and range over that ([example of this](https://stackoverflow.com/questions/73127947/helm-iterate-over-string-with-comma-separated-values)). By default I think range here will either glob things together or it might range over each character?